ethers-ext/src/ethers/signer.ts

```diff
@@ -102,9 +102,22 @@ export class Wallet extends EthersWallet {
     return tx;
   }
 
-  async populateTransaction(transaction: Deferrable<TransactionRequest>): Promise<TransactionRequest> {
-    let tx: TransactionRequest = await resolveProperties(transaction);
+  convertTxFromRLP(transaction: Deferrable<TransactionRequest> | string): any {
```

Comment:
Please mark this function `private`.

Reply:
Thanks 
I added https://github.com/klaytn/web3klaytn/pull/501/commits/4d2cb15937706928156030bf1bdb9c66714b8fd0.